main.js

```diff
@@ -0,0 +1,88 @@
+
+
+const initialState = [            //  declaring an array of arrays with "initialState"
+    ["5", "4", "3", "2", "1"],    //  This is the first peg at index[0], five discs on the peg to begin
+    [],                           //  This is the second peg at index[1], no discs to start (empty array)
+    [],                           //  This is the third peg at index[2], no discs to start (empty array)
+];                                //  Closing the array of arrays
+
+const game = {                         // declaring an object of game
+    board: [...initialState],          // key of board, the array ...initialState makes a copy of an object or an array content.  "Put these in there"
+    displayPegs: () => {               //  Create a function called displayPegs, arrow function format
+      console.log("gameboard:");       //  Console log of "gameboard" to indicate what the arrays represent
+      game.board.map((item) => {       //  game.board.map  -- used to CHANGE TO NEW VALUES.  "item" is an argument
+        console.log(item);             //  console.log the argument of item (from game.board.map)
```

Comment:
`Map` should be used to mutate/return an array - the way you're using it here is essentially how one would use `forEach`.